docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+  client:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Then add the build args in here:
```suggestion
      dockerfile: Dockerfile
      args:
        - RUBY_VERSION
```
We should be able to boot it with different ruby version via: `RUBY_VERSION=2.7 docker-compose up -d --build`